docs/docsite/rst/reference_appendices/release_and_maintenance.rst

```diff
@@ -151,6 +151,14 @@ Dates listed indicate the start date of the maintenance cycle.
      - End Of Life
      - Control Node Python
      - Target Python / PowerShell
+   * - 2.17
+     - | GA: 20 May 2024
+       | Critical: 04 Nov 2024
+     - | Security: 19 May 2025
+     - Nov 2025
+     - | Python 3.10 - 3.12
+     - | Python 3.7 - 3.12
+       | PowerShell TBD
```

Comment:
This might need a blank cell to fix the CI problem?

Reply:
I'll take a look later.